docs/Installation.md

```diff
@@ -99,7 +99,7 @@ Note: In case software-discovery-tool code is already checked out, do the follow
 
 (Alternatively, you can check with unittesting) <br />
 ```cd software-discovery-tool/src/tests``` <br />
-```pytest```
+```pytest -p no:warnings```
```

Comment:
Since some warnings are valuable, I think it's better if we let the current warnings come through, and maybe document why they exist, and why we can't or it's difficult to fix them?